apps/ejabberd/src/mod_admin_extra_stanza.erl

```diff
@@ -153,6 +160,16 @@ build_packet(message_headline, [Subject, Body]) ->
                       Stanza :: binary()) -> any().
 send_stanza_c2s(Username, Host, Resource, Stanza) ->
     C2sPid = ejabberd_sm:get_session_pid(Username, Host, Resource),
-    {ok, XmlEl} = exml:parse(Stanza),
-    p1_fsm:send_event(C2sPid, {xmlstreamelement, XmlEl}).
+    case C2sPid of
+        none ->
+            {bad_jid, io_lib:format("User ~s@~s/~s does not exist",[Username, Host, Resource])};
+        _ ->
+            case exml:parse(Stanza) of
+                {ok, XmlEl} ->
+                    p1_fsm:send_event(C2sPid, {xmlstreamelement, XmlEl}),
```

Comment:
Our `ejabberd_c2s` is no longer `p1_fsm` but `p1_fsm_old`. This shouldn't make any difference here but I'd use the correct module to call the process.